content-types/content-type-wallet-send-calls/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@xmtp/content-type-wallet-send-calls",
+  "version": "0.0.2",
```

Comment:
why not `0.0.1`?

Reply:
It was 0.0.1 earlier but as I was back-merging main I saw `content-type-transaction-reference` package version was [changed](https://github.com/HeyElsa/xmtp-js/commit/3e280789a7ee11f3af4497e848e2c9f3f6cb4aec#diff-d526eb570c5023cd812bb8b23a0dd1342cb49450fdadf5bed094a60b1b6de33cR3) from 2.0.0 to 2.0.1 and since this was copied mostly from that content-type I bumped the package version for this as well. Should this be 2.0.1 as well to be consistent with all the other content-types?